Add SOT-1210_LFPAK33

strombom opened the pull request:
SOT-1210, LFPAK33

https://assets.nexperia.com/documents/outline-drawing/SOT1210.pdf

![image](https://user-images.githubusercontent.com/875166/35456486-f0697ff0-02d5-11e8-8764-0f0ee71b210a.png)

![image](https://user-images.githubusercontent.com/875166/35456549-3190e676-02d6-11e8-9fb7-a92f6211117d.png)

![image](https://user-images.githubusercontent.com/875166/35456530-1ee32052-02d6-11e8-86f0-15a3d4707676.png)

3D Model pull request: https://github.com/KiCad/kicad-packages3D/pull/237

------------

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the footprint(s) you are contributing
- [x] An example screenshot image is very helpful 
- [x] If there are matching symbol or 3D model PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/KiCad/kicad-footprints?pullRequest=305) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/KiCad/kicad-footprints?pullRequest=305) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/KiCad/kicad-footprints?pullRequest=305) it.</sub>

strombom commented:
- Changed fab outline dimension to nominal
- Changed pin 1 marker to chamfer
- Removed pin number and copper from paste pads

![image](https://user-images.githubusercontent.com/875166/35464966-3772fd68-02f9-11e8-9375-4c23530cb8a3.png)
